README.md

```diff
@@ -67,15 +67,6 @@ settings page of your project on GitHub. See [Creating & using secrets].
 
 ### Trusted publishing
 
-> **IMPORTANT**: This functionality is in beta, and will not work for you
-> unless you're a member of the PyPI trusted publishing beta testers' group.
-> For more information, see [warehouse#12965].
-
-> **NOTE**: Trusted publishing is sometimes referred to by its
```

Comment:
I thought we still needed to have OIDC explanation. Plus, it's probably good for SEO. WDYT?

Reply:
Ah yep, I misread this -- I thought it said that PyPI still referred to OIDC in user facing docs, which it no longer does. 

I'll re-add this!

Reply:
Done!